controllers/platform/defaults.go

```diff
@@ -50,5 +51,25 @@ func ConfigureDefaults(ctx context.Context, c client.Client, p *operatorapi.Kogi
 		log.Log.Infof("Maven Timeout set to %s", p.Spec.BuildPlatform.Timeout.Duration)
 	}
 
+	if len(p.Spec.BuildPlatform.Registry.Address) == 0 {
+		updatePlatform(ctx, c, p)
+	}
+
 	return nil
 }
+
+func updatePlatform(ctx context.Context, c client.Client, p *operatorapi.KogitoServerlessPlatform) {
+	config := operatorapi.KogitoServerlessPlatform{}
```

Comment:
@radtriste  Because here https://github.com/kiegroup/kogito-serverless-operator/pull/144/files#diff-886bf47abbebc6c4ab83e53937b962ede2ef215bfb472d7e3b198c445bc49569R43 and here https://github.com/kiegroup/kogito-serverless-operator/pull/144/files#diff-886bf47abbebc6c4ab83e53937b962ede2ef215bfb472d7e3b198c445bc49569R48 the changes happens only when the address is empty, is the signal that the updatePlatform is needed

Reply:
Isn't recreated is the current retrieved with the c.Get

Reply:
yeah but then you are updating only specific fields...
My main worry is that if we add more fields modified, we forget to update here as well ...

Reply:
Ok, if is this your concern I'll update at the end without checks